### DISCUSSION TITLE
Add Chains: Avalanche and Avalanche Fuji

### DISCUSSION THREAD

**Discussion by taycaldwell** [wevm/wagmi#913] [Ideas]
Feature request to add chain(s):

- Avalanche
- Avalanche Fuji

[RainbowKit documents that these chains are already supported in wagmi](https://www.rainbowkit.com/docs/chains)

Is this already in progress? was it previously supported but removed?

I can open a PR to add it once I get the +1.

**Comment by jxom** [comment]
With the introduction of `0.9.x`, wagmi now comes with the `avalanche` & `avalancheFuji` chains built-in (re-exported from our [`@wagmi/chains` reference repository](https://github.com/wagmi-dev/references/tree/main/packages/chains)). Read more [here](https://wagmi.sh/react/chains).
